tests/src/main/scala/scalaz/tests/ACatenable1Tests.scala

```diff
@@ -19,8 +19,8 @@ object ACatenable1Tests {
   // these type aliases unnerve me with their presence
   type R[A, B] = ACatenable1[Map, A, B] => Map[A, B]
   type L[A, B] = Map[A, B] => ACatenable1[Map, A, B]
-  val foldF: ACatenable1[Map, ?, ?] ~~> Map = Forall2.of[R](_.fold)
-  val liftF: Map ~~> ACatenable1[Map, ?, ?] = Forall2.of[L](lift(_))
+  val foldBalancedF: ACatenable1[Map, ?, ?] ~~> Map = Forall2.of[R](_.foldBalanced)
```

Comment:
Do we have binatural transformations documented? That should definitely be on the list as we're adding more than a few instances here.